src/mlpack/core/data/pre_processing.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef MLPACK_CORE_DATA_DATA_PRE_PROCESSING_HPP
+#define MLPACK_CORE_DATA_DATA_PRE_PROCESSING_HPP
+
+#include <mlpack/core.hpp>
+#include <fstream>
+#include <vector>
```

Comment:
I think these two are already included by `core.hpp`. Also, if you can, try to include only `mlpack/prereqs.hpp` since it will include fewer other files and help keep compilation fast. :+1: